src/ui/class-block-editor.php

```diff
@@ -93,10 +99,12 @@ public function enqueue_block_editor_scripts() {
 
 		\wp_localize_script(
 			'duplicate_post_edit_script',
-			'duplicatePostLinks',
+			'duplicatePost',
 			[
 				'new_draft_link'             => $this->get_new_draft_permalink(),
 				'rewrite_and_republish_link' => $this->get_rewrite_republish_permalink(),
+				'rewriting'                  => \get_post_meta( $post->ID, '_dp_is_rewrite_republish_copy', true ) ? 1 : 0,
```

Comment:
Not sure why the `is_rewrite_and_republish_copy()` method from Permissions_Helper is not used...

Reply:
Because it didn't exist when this PR started 🙂